waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -115,3 +130,55 @@ method getAll*(db: WakuMessageStore, onData: message_store.DataProc): MessageSto
 proc close*(db: WakuMessageStore) = 
   ## Closes the database.
   db.database.close()
+
+
+proc migrate*(db: SqliteDatabase, path: string = MIGRATION_PATH, targetVersion: int64 = USER_VERSION): MessageStoreResult[bool] = 
+  ## compares the user_version of the db with the targetVersion 
+  ## runs migration scripts if the user_version is outdated (does not support down migration)
```

Comment:
Should we add a basic `markdown` file in docs folder re how migrations work? Ie something like:

"Migrations should work out of the box, however if you want to be extra sure take a backup of SQL DB first since we currently don't support downgrades of DB"

Reply:
Then we can point people to this every release we do with db migrations

Reply:
This doc can also mention to contributors to bump user_version etc

Reply:
Sure, good idea! I am working on a tutorial and will add it in the next PR.